api/controllers/CommunityController.js

```diff
@@ -1,3 +1,9 @@
+var Promise = require('bluebird'),
+  request = require('request'),
```

Comment:
Cool. Ok, I will have to look into that and aim for that in future. I'll add another commit changing it